src/commands/ping.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+	ping : {
+		name : 'ping',
+		args : false,
+		description : 'Ping in /commands',
+		execute(message, _data) {
```

Comment:
Any reason why this one is underscored and the others aren't?

Reply:
None, I'll fix that.

Reply:
Sounds good ^^